client/src/app/shared/models/event-management/committee.ts

```diff
@@ -12,6 +12,7 @@ export class Committee extends BaseModel<Committee> {
     public default_meeting_id: Id; // meeting/default_meeting_for_committee_id;
     public template_meeting_id: Id; // meeting/template_meeting_for_committee_id;
     public user_ids: Id[]; // (user/committee_ids)[];
+    public manager_ids: Id[]; // (user/manager_ids)[];
```

Comment:
This indicates a relation, which is right, but that relation is not registered in the `repositories/relation.ts`-file.
Also there are three things missing: the relation back from users to committees and the relations in the view-*-files of them.

Reply:
A second thing is, that the `manager_ids` has to be requested from the AU-service. So, the field has to be specified in the repo to the fieldsets. (You can follow them, too, to get request a specific fieldset, like only the `shortName`)

Reply:
This is not a database field! You cannot add non-database fields here

Reply:
Then we have to change this in the database

Reply:
most likely a design flaw, but we can get around that for now.